content/pandas/concepts/dataframe/terms/reset-index/reset-index.md

```diff
@@ -0,0 +1,160 @@
+---
+Title: '.reset_index()'
+Description: 'Resets the index of a DataFrame to be continuous' 
+Subjects: 
+  - 'Data Science'
+  - 'Pandas'
+Tags:
+  - 'Methods'
+  - 'Pandas'
+  - 'Data Science'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
+
+Through the course of exploratory analysis, and other data work, a DataFrame object will often be modified to clean and/or restructure the data. Through this work an index may become discontinuous or additional levels may be added or subtracted from the index. The **`.reset_index()`** method can be used to reestablish a continuous index as well as remove one or more unwanted levels.
+
+## Syntax
+
+```pseudo
+df = dataframe_value.reset_index()
+```
+
+The `.reset_index()` method provides the following parameters:
```

Comment:
Please reorder the parameters to match: level, drop, inplace, col_level, col_fill, allow_duplicates, names